### PR TITLE
minor english checks

### PR DESCRIPTION
**Pull request by chokomancarr**

### CODE REVIEW

**Comment by pandaman64** [comment]
*task local storage* is a term like *thread local storage*.
Perhaps hyphenating like *task-local storage* might be better?

File: src/coproduct.rs [hunk 1]
```diff
@@ -66,7 +66,7 @@ where
     /// Retrieve an effect and a sender from self if the type matches
     ///
     /// # Errors
-    /// If `self` holds an effect of a different type, this method returns an error
+    /// If `Self` holds an effect of a different type, this method returns an error
```
Comment (pandaman64) [inline src/coproduct.rs]: I intended to refer to the run-time value and its type, so I prefer `self` over `Self` here.

File: src/lib.rs [hunk 2]
```diff
@@ -227,11 +227,11 @@ pub trait Effectful {
...
     }
 
-    /// Run this computation to completion on the current thread
+    /// Run the computation to completion on the current thread
     ///
-    /// This method blocks the current thread while waiting for the progress of the computation
+    /// This method blocks the current thread while waiting on the progress of the computation
     ///
-    /// The effect type of this computation must be an empty set (never type) since there is no handler
+    /// The effect type of this computation must be an empty set (never typed) since there is no handler
```
Comment (pandaman64) [inline src/lib.rs]: Here, *never type* means the special uninhabited type `!` which denotes an empty set in the library.

**Comment by chokomancarr** [comment]
Thank you for the review
Reverted said mis-fixes

**Pull request merged**